oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -1146,7 +1145,7 @@ static int mca_spml_ucx_create_aux_ctx(void)
 
     rc = mca_spml_ucx_ctx_create_common(SHMEM_CTX_PRIVATE, &mca_spml_ucx.aux_ctx);
     if (OSHMEM_SUCCESS == rc) {
-        mca_spml_ucx.aux_ctx->spml_ucx = &mca_spml_ucx;
+        mca_spml_ucx.aux_ctx->synchronized_quiet = mca_spml_ucx_ctx_default.synchronized_quiet;
```

Comment:
should move it inside mca_spml_ucx_ctx_create_common

Reply:
will not work: there is array of pre-allocated objects which created without mca_spml_ucx_ctx_create_common call

Reply:
but mca_spml_ucx_ctx_create_common already sets other fields of *mca_spml_ucx.aux_ctx

Reply:
ok, moved